web/frontend/styles/casebook.scss

```diff
@@ -398,11 +398,35 @@ form.edit_content_resource, form.edit_content_section, form.edit_content_caseboo
     }
   }
 
-  /* TODO: this block is duplicated in TheResource.vue; factor out at some point */
+  .link-container :hover{
```

Comment:
Just a friendly reminder to include styles for keyboard-only users too 🙂

Reply:
Oops sorry 😅 will include that!